modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,13 +12,17 @@ const userLanguageId = Liferay.ThemeDisplay.getLanguageId();
 
 export function getLocalizableLabel(
 	creationLanguageId: Liferay.Language.Locale,
-	labels: LocalizedValue<string> | undefined,
+	labels: LocalizedValue<string> | string | undefined,
 	fallback?: string
 ) {
 	if (!labels) {
 		return fallback ?? '';
 	}
 
+	if (typeof labels === 'string') {
```

Comment:
Check the type in case the localized object has been processed by [the last change](https://github.com/liferay-frontend/liferay-portal/pull/3619/files#diff-75b3634a53391104d2407b67f552788627e4b2631e73cf0ae5d4a9b4dbf3fe61L132)